content/pages/disability-benefits/claims-appeal.md

```diff
@@ -153,15 +158,9 @@ After VBA has made a decision on your claim, if you disagree with the outcome, y
       </div>
     </li>
     <li>
-      <button class="usa-button-unstyled usa-accordion-button" aria-controls="dbq7">What happens if the Board makes a final decision and I disagree with it?</button>
+      <button class="usa-button-unstyled usa-accordion-button" aria-controls="dbq7">What happens if I disagree with the Board’s decision?</button>
       <div id="dbq7" class="usa-accordion-content">
-        <p>You can:</p>
-        <ul>
-          <li>File a new claim with your RO, <strong>or</strong></li>
-          <li>File a motion asking the Board to reconsider your appeal (there’s no time limit to file this motion), <strong>or</strong></li>
-          <li>File a motion asking the Board to review your appeal again because there was clear and unmistakable error in its decision (there’s no time limit to file this motion), <strong>or</strong></li>
-          <li>File a Notice of Appeal with the Court of Appeals for Veterans Claims (CAVC) within 120 days from the date of the decision by the Board (stamped on the first page of the decision). <a href="https://www.uscourts.cavc.gov/appeal.php">Learn how to file an appeal with the CAVC</a>.</li>
-        </ul>
+        <p>If you disagree with the Board’s decision, you can appeal to the Court of Appeals for Veterans Claims. You will need to hire a VA-accredited attorney to represent you, or you may represent yourself. You will need to file your Court appeal 120 days of the Board’s decision. <a href="https://www.uscourts.cavc.gov/appeal.php"> Learn how to file a Court appeal</a>.</p>
```

Comment:
Nit: If you're in here to edit something else, can you remove that extra space before "Learn" inside the `<a>`? It shouldn't be displayed, so don't feel like you have to make the change only to wait around for Jenkins to build again just because of this.

Reply:
Thanks @cvalarida, nice catch.